sys-fs/genfstab/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>marco@scardovi.com</email>
+		<name>Marco Scardovi</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
```

Comment:
```suggestion
	<maintainer type="person" proxied="proxy">
```
As I guessed, this returns a proper CI error, since it's implied none of the listed maintainers can commit :
```
sys-fs/genfstab
  MaintainerWithoutProxy: metadata.xml: proxied maintainers missing proxy dev/project: Marco Scardovi <marco@scardovi.com>, Matt Turner <mattst88@gentoo.org>
```
But looks like the pkgcheck ran on PR CI isn't up-to-date, or doesn't have this check enabled.

Reply:
Fixed

Reply:
Anyway qa reports the error correctly but i was too tired to fix it until today 😅